### PR TITLE
Add list organizations by mode CLI command

### PR DESCRIPTION
**Pull request by vijayshiyani**

### CODE REVIEW

File: packages/cli/src/commands/organizations/list.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  static description = 'List organization. There are three modes: validated, requested and member';
+
+  static flags = {
+    ...MeecoCommand.flags,
+    ...authFlags,
+    mode: _flags.string({
+      char: 'm',
+      default: 'validated',
+      required: false,
+      description:
```
Comment (zbarbuto) [inline packages/cli/src/commands/organizations/list.ts:19]: If there are 3 modes I think it's worth adding them as options:

```
    options: ['validated', 'requested', 'member'],          // only allow the value to be from a discrete set
```

https://oclif.io/docs/flags

Comment (vijayshiyani) [reply packages/cli/src/commands/organizations/list.ts:20]: done

File: packages/cli/src/configs/organizations-list-config.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+export class OrganizationsListConfig {
+  static kind = 'Organizations';
+
+  constructor(
+    public readonly templateName: string,
+    public readonly itemList: IOrganizationsListTemplate
+  ) {}
+
+  static encodeFromJSON(json: ListOrganizationsResponse1, metadata?: IOrganizationsListMetadata) {
```
[the remark this answers is not in the document]
Comment (vijayshiyani) [reply packages/cli/src/configs/organizations-list-config.ts]: changed thanks

**Comment by zbarbuto** [comment]
Conflicts in readme

**Pull request merged**